### ISSUE TITLE
AppHost creation fails under linux with server project

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno#13846]
The directory in question contains the following files:
`Dionysus.Server\bin\Debug\net7.0\Dionysus.Server\appsettings.Development.json`
`Dionysus.Server\bin\Debug\net7.0\Dionysus.Server\appsettings.json`
`Dionysus.Server\bin\Debug\net7.0\Dionysus.Server\Dionysus.Server.staticwebassets.runtime.json`

These files are duplicated and also exist in the parent directory with the rest of artifacts, where I think they _should_ be. They're both copied there as a part of the same build task. These files are also related to functionalities of appsettings configuration and WASM, so disabling those modules is not an option.

## Potential workaround

It may not be a .NET SDK issue, but this seems to be a feature that [can cause other problems](https://learn.microsoft.com/en-us/dotnet/core/install/macos-notarization-issues#native-apphost). You may try to disable it with:

```xml
<PropertyGroup>
  <UseAppHost>false</UseAppHost>
</PropertyGroup>
```

_Originally posted by @marcinkurek in https://github.com/unoplatform/uno/discussions/13808#discussioncomment-7115949_

**Comment by jeromelaban** [comment]
@marcinkurek this issue will be fixed by #13869, but the `UseAppHost` workaround works as well.

**Comment by marcinkurek** [comment]
Thank you so much! That's a perfectly acceptable workaround.
